src/app/src/main/java/com/example/bookmark/EditBookActivity.java

```diff
@@ -63,6 +64,20 @@ private void populateFields() {
         authorEditText.setText(book.getAuthor());
         isbnEditText.setText(book.getIsbn());
         descriptionEditText.setText(book.getDescription());
+        populatePhotograph();
+    }
+
+    private void populatePhotograph() {
+        if (book.getPhotograph().toString() != null) {
```

Comment:
Be careful not to call `toString()` on a potentially null object.
```suggestion
        if (book.getPhotograph() != null) {
```